src/MBC_UserImport_Source_Niche.php

```diff
@@ -11,6 +11,12 @@
 class MBC_UserImport_Source_Niche extends MBC_UserImport_BaseSource
 {
 
+  const WELCOME_EMAIL_NEW_NEW = 'mb-niche-welcome_new-new_v1-1-0';
+  const WELCOME_EMAIL_EXISTING_NEW = 'mb-niche-welcome_existing-new_v1-1-0';
+  const WELCOME_EMAIL_EXISTING_EXISTING = 'mb-niche-welcome_existing-existing_v1-1-0';
+  const MOBILE_COMMONS_SIGNUP = 207563; // Birthday Mail
```

Comment:
The opt-in path id is correct, but it refers to Shower Songs, not Birthday Mail. Can we update the text?